src/olympia/files/helpers.py

```diff
@@ -33,6 +33,21 @@
 LOCKED = 'locked'
 LOCKED_TIMEOUT = 60 * 5
 
+SYNTAX_HIGHLIGHTER_ALIAS_MAPPING = {
+    'xul': 'xml',
+    'rdf': 'xml',
+    'jsm': 'js',
+    'json': 'js',
+    'htm': 'html'
+}
+
+# See settings.MINIFY_BUNDLES['css']['zamboni/files'] for more details
```

Comment:
Fixed in https://github.com/mozilla/addons-server/pull/4600/commits/3a2667ba02c4a13ead3fec899a9c9bc2297f3445